client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -159,6 +160,8 @@ function MetadataListEditingDirective() {
         templateUrl: 'scripts/superdesk-authoring/metadata/views/metadata-terms.html',
         link: function(scope) {
 
+            subjectScope = scope;
```

Comment:
`subjectScope` doesn't seem to be used anywhere in the directive. It's also polluting the global namespace.